assembl/models/nlp.py

```diff
@@ -0,0 +1,382 @@
+import re
+
+import requests
+from sqlalchemy import (
+    Column,
+    ForeignKey,
+    Integer,
+    String,
+    Unicode,
+    Float,
+    Boolean,
+    UniqueConstraint,
+)
+from sqlalchemy.orm import relationship, backref, aliased
+from sqlalchemy.ext.declarative import declared_attr
+from sqlalchemy.ext.hybrid import hybrid_property
+from sqlalchemy.sql.functions import coalesce
+from sqlalchemy.sql.expression import case
+# from sqlalchemy.sql import func
+
+from . import Base, DiscussionBoundBase
+from .generic import Content
+from .langstrings import Locale, LangString, LangStringEntry
+from ..lib.logging import getLogger
+from .computation import Computation
+
+
+log = getLogger('assembl')
```

Comment:
Do you want to keep it to `assembl` or go nameless?

Reply:
added logging there as well. Note that this code is not currently used, but still a good idea to log.